sdcm/utils/health_checker.py

```diff
@@ -43,8 +43,8 @@ def check_nulls_in_peers(gossip_info, peers_details, current_node):
 
         is_target = current_node.print_node_running_nemesis(ip)
         message = f"Current node {current_node.ip_address}. Found nulls in system.peers for " \
-                  f"node {ip}{is_target} with status {gossip_info[ip]['status']} : " \
-                  f"{peers_details[ip]}"
+            f"node {ip}{is_target} with status {gossip_info[ip]['status']} : " \
+            f"{peers_details[ip]}"
```

Comment:
Is this (and following) whitespace changes forced by autopep8?

Reply:
Yes, I didn't touch it